### PR TITLE
Add jobs to make bors able to detect success or failure

### PR DESCRIPTION
**Pull request by rylev**
This is based on how [miri handles bors](https://github.com/rust-lang/miri/blob/6cf851f6d43bff1e8ac1421cbf3c6a8d20f89560/.github/workflows/ci.yml#L119-L140).

### CODE REVIEW

**Review by wesleywiser — APPROVED** [review]
LGTM is there anything else we need to do to get bors working?

**Comment by rylev** [comment]
My understanding of what we need to do is based on [this](https://github.com/rust-lang/rust-forge/blob/master/src/infra/docs/bors.md). I have PRs open for all the work items:
* https://github.com/rust-lang/homu/pull/154
* https://github.com/rust-lang/team/pull/627
* This PR

**Comment by camelid** [comment]
You might want to add a highfive config too, if you want auto-PR assignment.

**Comment by bjorn3** [comment]
Maybe it works of you close and re-open? The queue doesn't list any PR right now.

**Comment by camelid** [comment]
It seems like no GitHub Actions are actually running...?

**Comment by wesleywiser** [comment]
Maybe this needs to be adjusted? 

https://github.com/rust-lang/measureme/blob/59b79e5a0f5206741df6af66c66c6062263ee83a/.github/workflows/rust.yml#L3-L7

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -2,7 +2,10 @@ name: Rust
 
 on:
   push:
-    branches: [ master ]
+    branches: [ 
+      master,
+      auto    # used for bors
```
Comment (bjorn3) [inline .github/workflows/rust.yml:7]:
```suggestion
    auto,
    try
```

**Pull request merged**